webpack.config.js

```diff
@@ -1,58 +1,111 @@
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const path = require('path');
+/* eslint-disable @typescript-eslint/no-var-requires */
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+const webpack = require("webpack");
+const tailwindcss = require("tailwindcss");
+const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
+const InlineChunkHtmlPlugin = require("inline-chunk-html-plugin");
 
-const mode = process.env.NODE_ENV || 'development';
-const prod = mode === 'production';
+console.log("env", process.env.NODE_ENV);
+
+const mode = process.env.NODE_ENV || "development";
+const prod = mode === "production";
 
 module.exports = {
-    entry: './src/code.ts',
-    devtool: prod ? false : 'inline-source-map',
-    resolve: {
-        alias: {
-            svelte: path.resolve('node_modules', 'svelte')
-        },
-        extensions: ['.tsx', '.ts', '.js', '.svelte'],
-        mainFields: ['svelte', 'browser', 'module', 'main']
+  mode,
+  output: {
+    filename: "[name].js",
+    path: path.resolve(__dirname, "dist"), // Compile into a folder called "dist"
+    clean: true,
+    publicPath: "/",
+  },
+  entry: {
+    ui: "./src/main.js", // The entry point for UI code
+    code: "./src/code.ts", // The entry point for plugin code
+  },
+  devtool: prod ? false : "inline-source-map",
```

Comment:
you can make this false if you dont use this feature. i dont find it very helpful so leaving the choice upto you